### PR TITLE
runtest.sh: fix bug in deleting actual.txt from previous run #191

### LINKED ISSUE #191: runtest.sh: fix bug in deleting actual.txt from previous run
More specifically I am referring to [this part](https://github.com/se-edu/addressbook-level2/blob/2d5071d43d423edf521cae55e446d2647021c9b5/test/runtests.sh#L9).

Clearly, `actual.txt` should be a file and its existence should be tested using `-e` not `-d` and its directory should be `./` not `../`

This bug is not noticed because `[ ! -d "../actual.txt" ]` will return true, the same as the correct command `[-e ./actual.txt]`. So the `./actual.txt` from previous run will still be removed and no strange behaviour is observed.

However, in travis, where each time a new environment is created, the bug is noticed because Travis reports ```rm: cannot remove `actual.txt': No such file or directory```

### PR DESCRIPTION
**Pull request by chao1995**
Fixes #191 

Ready for review.

### CODE REVIEW

**Comment by CanIHasReview** [comment]
# v1

@chao1995 submitted v1 for review.

* [v1 1/1] [runtest\.sh: fix bug in deleting actual\.txt from previous run](https://github.com/se-edu/addressbook-level2/pull/192/commits/4b458debcd6f0d2c784b08784468f94002512022)

([:books: Archive](https://github.com/se-edu/addressbook-level2/compare/2d5071d43d423edf521cae55e446d2647021c9b5...4b458debcd6f0d2c784b08784468f94002512022))
<details>
<summary>Checkout this PR version locally</summary>

```shell
git fetch https://github.com/se-edu/addressbook-level2.git refs/pr/192/1/head:BRANCHNAME
```
where `BRANCHNAME` is the name of the local branch you want to fetch this PR to.

</details>

**Comment by chao1995** [comment]
> It deletes 'actual.txt' only when it does not exist (with "!").

I modify it to `tries to delete` because when the file doesn't exist, we can't really delete it.

> for deleting previous run's output

Modify to ```for deleting './actual.txt' from previous run:```. Make it clear the output is "./actual.txt"

**Comment by CanIHasReview** [comment]
# v2

@chao1995 submitted v2 for review.

* [v2 1/1] [runtest\.sh: fix bug in deleting actual\.txt from previous run](https://github.com/se-edu/addressbook-level2/pull/192/commits/1cc5741f828d069f3b88614c6b647b8b9a0ded1a)

([:books: Archive](https://github.com/se-edu/addressbook-level2/compare/2d5071d43d423edf521cae55e446d2647021c9b5...1cc5741f828d069f3b88614c6b647b8b9a0ded1a)) ([:chart_with_upwards_trend: Interdiff between v1 and v2](https://github.com/se-edu/addressbook-level2/compare/5c27946a17abbd6c5744469037d5a8c3fad5fd2f~1...5c27946a17abbd6c5744469037d5a8c3fad5fd2f))
<details>
<summary>Checkout this PR version locally</summary>

```shell
git fetch https://github.com/se-edu/addressbook-level2.git refs/pr/192/2/head:BRANCHNAME
```
where `BRANCHNAME` is the name of the local branch you want to fetch this PR to.

</details>

File: test/runtests.sh [hunk 1]
```diff
@@ -7,7 +7,7 @@ then
 fi
 
 # delete output from previous run
-if [ ! -d "../actual.txt" ]
+if [ -e "./actual.txt" ]
```
Comment (pyokagan) [inline test/runtests.sh]: I think the `../` is correct since this script is meant to be run in the `test` directory?

Comment (pyokagan) [inline test/runtests.sh]: Oh right, it's precisely because this needs to be run in the `test` directory. Whoops.

Comment (pyokagan) [inline test/runtests.sh]: As an aside, this whole block is probably better served with a straightforward `rm -f actual.txt`.

**Pull request merged**